src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--fri calendar--month-length--28">
+      <div class="calendar__day calendar__day--1"></div>
+      <div class="calendar__day calendar__day--2"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```
implement this task without those classes

Reply:
i dont uderstand how to make it this way

Reply:
Still not fixed from the previous review, you need to remove it everywhere
<img width="779" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/64750958/42d409fb-f657-407b-96f8-e936162224aa">